Gopkg.lock

```diff
@@ -33,7 +33,7 @@
   version = "v10.8.1"
 
 [[projects]]
-  digest = "1:38f778cf0ee78a06a5c80a514ac70792c662402e0cf342af92d21146ee161fb6"
```

Comment:
I see unrelated projects getting their digest updated, revert these changes to keep only changes related to your PR.

Reply:
updated